### ISSUE TITLE
Add Item - The 'add input' button is ambiguous

### ISSUE THREAD

**Issue by katknow** [omeka/Omeka#1146]
"Description: Upon testing this page, it was found that there are many 'add input' buttons throughout the 'Dublin Core' section and others. These buttons don't totally convey what the button is for. Currently, the button is labelled as 'Add Input'. Which does not convey the type of input being added. The buttons could be confused for ""Add text Input"" (submit) buttons after filling out the fields. It would be better if the button were labelled 'Add Input Field', 'Add Input Row', or similar. 

Expected Results: All interactable elements should be specific in conveying what the button will do.

Actual Results: The 'add input' button is somewhat ambiguous to users."

2.4.6 Headings and Labels; Level AA; WCAG 2.0

**Comment by kimisgold** [comment]
The "Add Input" buttons now have labeled group containers contextualizing them within a form field. This can be tested on the [`a11y/form-field-groups`](https://github.com/omeka/Omeka/tree/a11y/form-field-groups) branch.
